src/app/loader.ts

```diff
@@ -56,12 +58,33 @@ export function loadFiles(options?: LoadFilesOptions): Promise<LoadFilesResult>
         { prefetch: true }
     );
 
+    const languageConfig = isLanguageWindow(window) ?
+        window.language :
+        { locale: 'en', locales: {}, translations: {} };
+
     return Promise.all([
+        getDefaultTranslations(languageConfig.locale),
         scripts,
         stylesheets,
     ])
-        .then(() => {
-            const { renderCheckout, renderOrderConfirmation } = (window as any)[LIBRARY_NAME];
+        .then(([defaultTranslations]) => {
+            // We have to use `any` here because we can't define an interface with a variable defined by Webpack
+            const appExport = (window as any)[LIBRARY_NAME];
```

Comment:
```tsx
type KeyInWindow<T extends string> = Record<T, unknown> & Window; 

function isKeyInWindow<T extends string>(win: Window, key: T): win is KeyInWindow<T> {
    return key in win;
}
```

```tsx
if (!isKeyInWindow(window, LIBRARY_NAME)) {
  throw new Error(`App ${LIBRARY_NAME} not available in window`);
}

const appExport = window[LIBRARY_NAME];
```

Considering we, most of the time, just throw an error if the key is not available it might be better to use `asserts` 🤷